dev/devicelab/manifest.yaml

```diff
@@ -221,6 +228,13 @@ tasks:
 #    required_agent_capabilities: ["has-ios-device"]
 #    flaky: true    
 
+  e2e_ios:
+    description: >
+      Runs end-to-end Flutter tests on iOS.
+    stage: devicelab_ios
+    required_agent_capabilities: ["has-ios-device"]
+    flaky: true
```

Comment:
Yegor's suggestion was to mark new tests flaky until we've seen them run a few rounds, then mark unflaky.

In terms of whether they've been flaky in local runs -- no.